src/options-pages/service.cpp

```diff
@@ -47,13 +47,13 @@ Service::Service(Settings::ServicePtr settings)
                     Utils::Icon::Tint));
 }
 
-bool Service::matches(const QString &searchKeyWord) const
+bool Service::matches(const QRegularExpression &regexp) const
 {
     const QStringList keywords{"asn1scc", "daemon", "asn1.exe", "asn1", "asn.1", "acn"};
     for (const auto &keyword : keywords)
-        if (keyword.contains(searchKeyWord, Qt::CaseInsensitive))
+        if (keyword.contains(regexp.pattern(), Qt::CaseInsensitive))
```

Comment:
should be other way around

also (as this is a clearly code duplication, when I have to write the same comment 3 times) - try extracting some utility method or check how other plugins do it, maybe there's something available already there